schema/data-model-schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "ProductFootprint",
+  "title": "ProductFootprint_for_AnyValue",
```

Comment:
It would be great if the exported JSON Schema type name would equal the ones used in the Tech Specs. 
Do you have an idea how to achieve this?